Nmaksimchicov/1.HTML/3.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<HTML>
+    <HEAD>
+        <META charset="utf-8">
+        <TITLE>Phython 02</TITLE>
+    </HEAD>
+<!--начало документа -->
+    <BODY>
+	<h1><strong>HOW-TO: Программа на Python, Часть 2</strong></h1>
+<ul>
+	<li><em>Программа на Python — часть 1 </em></li>
+	<li><em>Программа на Python — часть 2</em></li>
+	<li><em>Программа на Python — часть 3</em></li>
+</ul>
+<h2><strong>Содержание</strong></h2>
+<h2><strong>HOW-TO: Программа на Python, Часть 2</strong></h2>
+<ul>
+	<li><ins>Списки</ins></li>
+	<li><ins>Комментарии</ins></li>
+</ul>
+<hr width="100%" color="##000000" align="left" size="1">
+<p>Впрошлом уроке мы рассмотрели простую программу, которая использовала raw_input для получения ответа пользователя, простые типы переменных и простой цикл «for». В этом уроке мы рассмотрим ещё больше типов переменных и напишем ещё больше программ.</p>
+<h2><strong>Списки</strong></h2>
+<p>Давайте рассмотрим другой тип переменных, который называется списки. В других языках программирования они называются массивами. Если мы обратимся к аналогии с коробками для обуви, то массив (список) представляет собой несколько склеенных друг с другом коробок с одинаковыми вещами. Например, в одной коробке лежат вилки, в другой — ножи, в третьей — ложки. Посмотрим на простой список— список месяцев. Мы запрограммируем его так:</p>
+<pre>months = 
+['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec']</pre>
+<p>Чтобы создать список, мы заключили все значения в квадратные скобки ( '[' и ']' ). Мы назвали наш список 'months'. Чтобы использовать список, мы дадим команду print months[0] или months[1] (которые выведут на экран 'Jan' или 'Feb'). Помните: счет начинается с нуля. Чтобы найти длину списка, можно использовать такую функцию:</p>
+<pre>print len(months)</pre>
+<p>которая вернёт 12.</p>
+<p>Ещё один пример списков — категории в поваренной книге. Например…</p>
+<pre>categories = ['Main dish','Meat','Fish','Soup','Cookies']</pre>
+<p>Таким образом, в <strong>categories[0]</strong> хранится <em>'Main dish'</em>, а в <strong>categories[4]</strong> — <em>'Cookies'</em>. Всё очень просто. Я уверен, вы сможете придумать много других примеров использования списков.</p>
+<hr width="100%" color="##000000" align="left" size="1">
+<h2>Комментарии</h2>
+<p>Далее, мы должны обсудить комментарии. Комментарии важны по нескольким причинам. Они не только говорят вам или другим людям, что вы пытаетесь делать, но и напомнят вам через полгода, что же вы хотели сделать. Когда вы будете писать много программ, комментарии станут важными. Также при их помощи вы заставите Python игнорировать какие-либо части кода. Чтобы закомментировать строку, поставьте перед ней знак '#'. Например:</p>
+<p>#</p> 
```

Comment:
It is code

Reply:
could you be more specific.

Reply:
Why do you use paragraph tag for code section?